app/viewUtils/ReportingFrequencyViewUtils.scala

```diff
@@ -33,27 +35,60 @@ class ReportingFrequencyViewUtils @Inject()()(
 ) extends FeatureSwitching {
 
 
-  def itsaStatusString(itsaStatus: ITSAStatus)(implicit messages: Messages): Option[String] = {
+  def itsaStatusString(itsaStatus: ITSAStatus)(implicit messages: Messages, user: MtdItUser[_]): Option[String] = {
+    if(isEnabled(OptInOptOutContentUpdateR17)){
+      itsaStatus match {
+        case Mandated => Some(messages("reporting.frequency.table.mandated.r17"))
+        case Voluntary => Some(messages("reporting.frequency.table.voluntary.r17"))
+        case Annual => Some(messages("reporting.frequency.table.annual.r17"))
+        case _ => None
+      }
+    }else{
+      itsaStatus match {
+        case Mandated => Some(messages("reporting.frequency.table.mandated"))
+        case Voluntary => Some(messages("reporting.frequency.table.voluntary"))
+        case Annual => Some(messages("reporting.frequency.table.annual"))
+        case _ => None
+      }
+    }
+  }
+
+  private def isUsingMTD(itsaStatus: ITSAStatus)(implicit messages: Messages): Option[String] = {
     itsaStatus match {
-      case Mandated => Some(messages("reporting.frequency.table.mandated"))
-      case Voluntary => Some(messages("reporting.frequency.table.voluntary"))
-      case Annual => Some(messages("reporting.frequency.table.annual"))
+      case Mandated => Some(messages("reporting.frequency.table.MTD.voluntary"))
```

Comment:
case Mandated => Some(messages("reporting.frequency.table.MTD.voluntary"))

is this meant to be the voluntary message or should it be "Required"?

Reply:
same with 

reporting.frequency.table.MTD.annual